### ISSUE TITLE
Update date in the copyright notice in all source files

### ISSUE THREAD

**Issue by sequba** [handsontable/hyperformula#1041]
### Description

Change
```
/**
 * @license
 * Copyright (c) 2021 Handsoncode. All rights reserved.
 */
```
to
```
/**
 * @license
 * Copyright (c) 2022 Handsoncode. All rights reserved.
 */
 ```

### Links

- https://handsoncode.slack.com/archives/C031GBX62SE/p1659367191186529 (internal discussion)

**Comment by sequba** [comment]
I accidentally [committed](https://github.com/handsontable/hyperformula/commit/52b284f0d2e79fed96adf9ecd71965abb7921467) it directly to `develop`. I updated the repository branch protection rules so it won't happen again.

**Comment by sequba** [comment]
Closing as released in version [2.1.0](https://github.com/handsontable/hyperformula/releases/tag/2.1.0).
